assets/js/components/notifications/SwitchGA4DashboardViewNotification.js

```diff
@@ -59,7 +64,20 @@ export default function SwitchGA4DashboardViewNotification() {
 	const { setValue } = useDispatch( CORE_UI );
 	const { dismissItem } = useDispatch( CORE_USER );
 	const { setDashboardView, saveSettings } = useDispatch( MODULES_ANALYTICS );
-	const handleCTAClick = () => {
+
+	const handleOnView = useCallback( () => {
+		trackEvent(
+			`${ viewContext }_ga4-display-notification`,
```

Comment:
This value is used several times, how about we extract an `eventCategory` variable similar to other uses?